nearlib/test/integration.test.js

```diff
@@ -18,6 +18,7 @@ const nearClient = new NearClient(test_key_store, localNodeConnection);
 const account = new Account(nearClient);
 const nearjs = new Near(nearClient);
 const TEST_MAX_RETRIES = 10;
+const TRANSACTION_COMPLETE_MAX_RETRIES = 100;
```

Comment:
why retrying 100 times is needed? Does this mean we are retrying too fast?

Reply:
@janedegtiareva